src/WebUI/WorkLoads/DeploymentsTable.tsx

```diff
@@ -252,40 +276,49 @@ export class DeploymentsTable extends BaseComponent<IDeploymentsTableProperties,
         return BaseKubeTable.renderTableCell(rowIndex, columnIndex, tableColumn, itemToRender);
     }
 
-    private static _renderImageCell(rowIndex: number, columnIndex: number, tableColumn: ITableColumn<IDeploymentReplicaSetItem>, deployment: IDeploymentReplicaSetItem): JSX.Element {
-        const textToRender: string | undefined = deployment.image;
-        const itemToRender = BaseKubeTable.renderColumn(textToRender || "", BaseKubeTable.defaultColumnRenderer, colDataClassName);
+    private _renderImageCell=(rowIndex: number, columnIndex: number, tableColumn: ITableColumn<IDeploymentReplicaSetItem>, deployment: IDeploymentReplicaSetItem): JSX.Element=> {
+        const textToRender: string = deployment.imageDisplayText;
+        const imageId: string = deployment.imageId;
+        // HardCoding hasImageDetails true for the time being, Should change it once we integrate with ImageService
+        //const hasImageDetails: boolean = this._hasImageDetails && this._hasImageDetails.hasOwnProperty(firstImageName) ? this._hasImageDetails[firstImageName] : false;
+        const hasImageDetails = true;
+        const itemToRender = hasImageDetails ?
+            (<Link
```

Comment:
we may need to change this

Reply:
Will revisit Links once your PR is checked in